devel/parallel-hashmap/Portfile

```diff
@@ -16,26 +16,22 @@ maintainers             {@barracuda156 gmail.com:vital.had} openmaintainer
 description             A family of header-only, very fast and memory-friendly hashmap and btree containers
 long_description        {*}${description}
 homepage                https://greg7mdp.github.io/parallel-hashmap
-checksums               rmd160  2562cd557a276f73ac823465b4781d6bb2efc9ed \
-                        sha256  0cc203144321924cfbfcc401f42d8204c0dd24e2760c7a1c091baa16d9777c08 \
-                        size    2047800
+checksums               rmd160  a37046afe248c835966a3b66e5d242b5f3b8dd30 \
+                        sha256  4f462f51a3468166ea4cf87c80e001dc1999093264cf55cbda3492ca39a7730b \
+                        size    2059086
+
 github.tarball_from     archive
+depends_build-append    port:gtest
 
-platform darwin {
-    if {${os.major} < 14} {
-        # Lion+ (with Xcode 4.1+) have git; earlier need to bring their own.
-        # On 10.9 git fetch fails with ssl error.
-        depends_build-append    port:git
-        git.cmd                 ${prefix}/bin/git
-    }
-}
+patchfiles              gtest-patch.diff
 
-compiler.cxx_standard   2011
+compiler.cxx_standard   2014
 compiler.thread_local_storage yes
 
 configure.args-append   -DPHMAP_INSTALL=ON \
                         -DPHMAP_BUILD_TESTS=ON \
                         -DPHMAP_BUILD_EXAMPLES=OFF
+configure.ldflags-append -pthread
```

Comment:
Last thing – sorry, missed this – is this actually needed on some macOS version? It should be set by CMake if not by compiler to begin with.

Reply:
P. S. And if it _is_ needed, then it is worth also reporting it to upstream so that it can be fixed by next release. I only tried on 10.6 and 10.15, neither needed manual `-pthread`.